builtin/providers/aws/resource_aws_network_interface.go

```diff
@@ -200,6 +199,14 @@ func resourceAwsNetworkInterfaceDetach(oa *schema.Set, meta interface{}, eniId s
 	return nil
 }
 
+func convertToAwsStringSlice(s []interface{}) []*string {
```

Comment:
We have a method just like this! It's `expandStringList`:

- https://github.com/hashicorp/terraform/blob/master/builtin/providers/aws/structure.go#L430

I'll go ahead and merge as-is and then replace myself, so you don't need to be troubled